src/Compilers/CSharp/Test/Syntax/Parsing/LocalFunctionParsingTests.cs

```diff
@@ -568,6 +568,79 @@ void M()
                 Diagnostic(ErrorCode.ERR_EOFExpected, "}").WithLocation(8, 1));
         }
 
+        [Fact]
+        [WorkItem(38801, "https://github.com/dotnet/roslyn/issues/38801")]
+        public void LocalFunction_NoBody()
+        {
+            UsingTree(@"
```

Comment:
>UsingTree [](start = 12, length = 9)

What is the purpose of these tests if we are not asserting the shape of the syntax tree?